src/devices/Is31fl3731/samples/Program.Matrix.cs

```diff
@@ -0,0 +1,205 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Device.I2c;
+using System.Linq;
+using System.Threading;
+using Iot.Device.Display;
+
+// For 16x9 matrix
+// https://www.adafruit.com/product/2974
+using I2cDevice i2cDevice = I2cDevice.Create(new I2cConnectionSettings(busId: 1, Backpack16x9.DefaultI2cAddress));
+Backpack16x9 matrix = new(i2cDevice);
+// For 16x8 matrix Charlieplex bonet
```

Comment:
maybe itroduce an empty line between each block of matrix? you can as well use 4 slash or dashes or anything to make it more redable

Reply:
Good call. Done.